processing/src/main/java/io/druid/query/search/search/ConciseBitmapDecisionHelper.java

```diff
@@ -0,0 +1,12 @@
+package io.druid.query.search.search;
+
+public class ConciseBitmapDecisionHelper extends SearchQueryDecisionHelper
+{
+  private static final double LOW_FILTER_SELECTIVITY_THRESHOLD = 0.99;
+  private static final int LOW_CARDINALITY_THRESHOLD = 5000;
```

Comment:
Could you please briefly describe, in a comment, where these numbers came from? (if you decide to keep using them rather than the cost based approach)